src/test/java/CarRaceGameTest.java

```diff
@@ -0,0 +1,71 @@
+import model.Car;
+import model.CarRaceGame;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class CarRaceGameTest {
+    @Test
+    @DisplayName("주어진 자동차를 이용해 경주를 진행할 경우 올바르게 이동 거리가 반영된 자동차 인스턴스 목록을 반환해야 한다.")
+    void should_GetCarList_AfterRounds() {
+        Car aliceCar = new Car("Alice", new TestRandomNumberGenerator((Arrays.asList(0, 2))));
+        Car bobCar = new Car("Bob", new TestRandomNumberGenerator((Arrays.asList(8, 3))));
+        Car carlCar = new Car("Carl", new TestRandomNumberGenerator((Arrays.asList(4, 5))));
+        List<Car> cars = new ArrayList<>(Arrays.asList(aliceCar, bobCar, carlCar));

```

Comment:
테스트별로 자동차를 생성하는 코드가 많이 중첩되는데 차라리 이걸 함수로 따로 빼는건 어때?

```suggestion
    private CarRaceGame setupRaceWithCars(List<Car> cars) {
    return new CarRaceGame(cars);
}

private List<Car> createTestCars(List<List<Integer>> randomValues, List<String> names) {
    List<Car> cars = new ArrayList<>();
    for (int i = 0; i < names.size(); i++) {
        cars.add(new Car(names.get(i), new TestRandomNumberGenerator(randomValues.get(i))));
    }
    return cars;
}

@Test
@DisplayName("")
void should_ReturnWinningCarNames() {
    List<Car> cars = createTestCars(
        Arrays.asList(Arrays.asList(1, 5, 8), Arrays.asList(0, 9, 3), Arrays.asList(2, 6, 7)),
        Arrays.asList("Alice", "Bob", "Carl")
    );
```

Reply:
좋은 아이디어에요, 덕분에 코드가 훨씬 간결해졌어요 고맙습니다 👍🏻